driver/compenv.ml

```diff
@@ -15,7 +15,7 @@
 
 open Clflags
 
-exception Exit_with_status of int
+exception Exit_with_status = Clflags.Exit_with_status
```

Comment:
Hm. I see why you need this, but I liked the way this logic was self-contained within `Compenv` (this stuff was recently introduced in #9688 ). One other option would be to try to move `Clflags.parse_arguments` to `Compenv`. Is there an obvious reason why this would not work? Otherwise, could you maybe give it a try, and see if you like it more than the current proposal?

Reply:
moved parse_arguments to from Clflags to Compenv, however i had to make create_usage_msg function public from Clflags because it was used by print_arguments, should i move print_arguments to Compenv too in order to remove additional public declaration from Clflags?